app/helpers.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+use Gladiator\Models\User;
+use Gladiator\Services\Northstar\Exceptions\NorthstarUserNotFoundException;
+
+if (! function_exists('matchEmailDomain')) {
```

Comment:
You probably don't need this in a "app" helpers directory, since you wouldn't ever want to accidentally use an existing global function here.